flask_resty/sorting.py

```diff
@@ -128,36 +129,67 @@ def get_request_field_orderings(self, view):
 class Sorting(FieldSortingBase):
     """A sorting component that allows the user to specify sort fields.
 
-    For example, to allow users to sort by `name` and/or `date`, specify the
-    following in your view::
+    For example, to allow users to sort by `title` and/or `content_length`,
+    specify the following in your view::
 
-        sorting = Sorting('name', 'date')
+        sorting = Sorting(
+            'title',
+            content_length=sql.func.length(Post.content)
+        )
 
-    One or both of `name` or `date` can be formatted in the `sort_arg` request
-    parameter to determine the sort order. For example, users can sort requests
-    by `name` ascending and `date` descending by making a ``GET`` request to::
+    One or both of `title` or `content_length` can be formatted in the
+    `sort_arg` request parameter to determine the sort order. For example,
+    users can sort requests by `name` ascending and `date` descending by
+    making a ``GET`` request to::
 
-        /api/widgets?sort=name,-date
+        /api/post?sort=title,-content_length
```

Comment:
nitpick: We typically use plural nouns, and `comments` is less overloaded than `posts`.

```suggestion
        /api/comments/?sort=title,-content_length
```